packages/docs/content/patterns/utility-actions.mdx

```diff
@@ -0,0 +1,34 @@
+---
+title: Utility Actions
+intro: Utility actions for a webpage including print, email, RSS and more.
+---
+
+import { TextField } from '@cmsgov/design-system';
+
+## Guidance
+
+### When to use
+
+- If you would like to give the user the option to print or email a page.
+- If you would like to give the user the option to enable RSS.
+- These "utility actions" are considered [ghost buttons](/components/button/#ghost-button) for actions available that do not take priority over a primary or secondary CTAs (e.g., print, email, RSS).
+
+### When to consider alternatives
+
+- If an action (e.g. printing or emailing) is a critical call-to-action on a webpage, consider using a solid or outlined main button.
+- If the utility action is opening a link versus triggering an action, please reference [guidance on links](/foundation/typography/links).
+- Consider carefully which Utility Actions are used on mobile since space is a premium.
+
+### Usage
+
+- This pattern uses a [ghost button](/components/button/#ghost-button) component with a left icon boolean.
```

Comment:
This "left icon boolean" must be a reference to Figma, because there's no such analog in code. Maybe we should be explicit that this is a design reference.

For the code part, are we going to show an example of what it would look like? A storybook story or something?

Reply:
Good call on stating that this is a ref to Figma. There will be code via Storybook, that is in development hence the placeholder to the TextField component.